tests/e2e/gateway/01-install.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+   name: minio-test
+stringData:
+  endpoint: http://minio.minio.svc:9000
+  bucket: tempo
+  access_key_id: tempo
+  access_key_secret: supersecret
+type: Opaque
+---
+apiVersion: tempo.grafana.com/v1alpha1
+kind: Microservices
+metadata:
+  name: foo
+spec:
+  images:
+    tempoGateway: pavolloffay/observatorium-api:001
+  template:
+    gateway:
+      enabled: true
+  storage:
+    secret: minio-test
+  storageSize: 200M
+  tenants:
+    mode: static
+    authentication:
+      - tenantName: test-oidc
+        tenantId: test-oidc
+        oidc:
+          secret:
+            name: test-oidc
+          issuerURL: https://dex.klimlive.de/dex
```

Comment:
Could we use something ownered by the team or something that can be started in the test environment?

Reply:
We will do in future tests, here it actually doesnt matter. Since its not used.

Reply:
So: since it is not used, you can remove it.

Reply:
Does it still work when `dex.klimlive.de` is replaced with `localhost`?